src/renderer/init.js

```diff
@@ -83,11 +83,11 @@ async function init() {
 
   const isMainWindow = remote.getCurrentWindow().name === "MainWindow";
 
-  if (initialSettings.hasPassword) {
-    store.dispatch(lock());
-  } else {
-    const accounts = await getKey("app", "accounts", []);
+  const accounts = await getKey("app", "accounts", []);
```

Comment:
@gre Due to the changes from https://github.com/LedgerHQ/lld-v2/pull/188/files#diff-70f2e46b1f570eb03352ea45e36240e2R16-R20 we no longer need to try/catch to infer this locked state